web/src/app/util/CountDown.tsx

```diff
@@ -29,32 +29,37 @@ export function formatTimeRemaining(
   if (weeks) {
     const numWeeks = timing / 604800 // There are 604800s in a week
     timing = timing - numWeeks * 604800
-    if (numWeeks) timeString += numWeeks + ' Week' + (numWeeks > 1 ? 's ' : ' ')
+    if (numWeeks)
+      timeString += Math.trunc(numWeeks) + ' Week' + (numWeeks > 1 ? 's ' : ' ')
```

Comment:
Should be truncated at the top: `const numWeeks = Math.trunc(timing / 604800)`

Otherwise the `if` statement and `timing = timing -` lines get the wrong value.